ts/live-compositor/src/api.generated.ts

```diff
@@ -552,6 +590,22 @@ export type Component =
        * effect if the previous scene already contained a `Rescaler` component with the same id.
        */
       transition?: Transition | null;
+      /**
+       * (**default=`0.0`**) Radius of a rounded corner.
+       */
+      border_radius?: number | null;
```

Comment:
not part of this PR, but you could add properties from this diff too new docs if we don't plan to version old compositor releases